vyper/venom/venom_to_assembly.py

```diff
@@ -434,13 +434,6 @@ def _generate_evm_for_instruction(
             # the same variable, however, before a jump that is not possible
             self._stack_reorder(assembly, stack, list(target_stack))
 
-        if inst.is_commutative:
```

Comment:
This creates regression for me (if I reintroduce it it will create smaller codesize on curve swap)

Reply:
indeed

Reply:
interestingly, i found that "always flip if is_commutative" actually performs better for curve swap than any heuristic i came up with.